l1-contracts/src/core/interfaces/messagebridge/IInbox.sol

```diff
@@ -7,7 +7,7 @@ import {DataStructures} from "../../libraries/DataStructures.sol";
 /**
  * @title Inbox
  * @author Aztec Labs
- * @notice Lives on L1 and is used to pass messages into the rollup, e.g., L1 -> L2 messages.
+ * @notice Lives on L1 and is used to pass messages into the rollup from L1.
```

Comment:
Did this change just to try to retrigger CI and found the e.g. there to be a bit misleading as it kinda implies that there could be other kind of messages in the inbox.